### PR TITLE
tools: shorten directory path

### PR DESCRIPTION
**Pull request by converge**

### CODE REVIEW

**Comment by fidencio** [comment]
Hey @converge, thanks for providing your first contribution to the kata-containers project!  We appreciate it and we hope this is the first of many! :-)

There are some guidelines one should follow to contribute to the project, you can find that here: https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md

In a really short form, you'll need:
* Commit your patches using `git commit --signoff`, due to the certificate of origin;
* The commit message must include a `Fixes: #NNN`, where NNN is the number of the issue you're fixing (so, in your case, it'd be: `Fixes: #2354`
* The commit message should explain the problem you're fixing.

Here's one example of what you can use as a guideline: https://github.com/kata-containers/kata-containers/commit/3fe6695b0a7f0f1a132efa9b2ebd11f5e059f270

Please, update your patch, force push the changes into the same branch, and let us know so we can run the tests and have it reviewed & merged.

Thanks for your time and for the interest in the project!

**Comment by converge** [comment]
@fidencio tks for the prompt reply and the information =)

I guess it's correct now.

**Comment by fidencio** [comment]
> @fidencio tks for the prompt reply and the information =)
> 
> I guess it's correct now.

Almost there.
I'd recommend this:
```
kata-deploy: shorten directory path

Long file paths are difficult to read, this change adds a new readonly
variable to shorten the full file path of the static build folder files.

Fixes: #2354

Signed-off-by: Joao Vanzuita <joaovanzuita@me.com>
```

Thanks!

**Comment by converge** [comment]
@fidencio sounds good. one more review pls =)

File: tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh
Comment (fidencio) [inline tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh:19]: It took me some mental effort to ensure that `${repo_root_dir}/tools/packaging/static-build` was the same as `${script_dir}/../../static-build`.  But this is way more on us than on you, @converge. :-)

**Review by fidencio — APPROVED** [review]
lgtm, thanks @converge.

Personally I'd have used `${repo_root_dir}/tools/packaging/static-build` rather than `${script_dir}/../../static-build`, but that's me and unless someone else raises the same point I think it's good as it is.

**Comment by converge** [comment]
@fidencio tks for the review, just did another update following our chat.

**Comment by fidencio** [comment]
> @fidencio tks for the review, just did another update following our chat.

@converge thanks a lot for the change.
Would be possible to squash the second commit into the first one?  I know, some repos do that while merging, but that's not the case for this project.

Anyways, after having this squashed, I think it's perfect and we can merge it after a second review / approval.

Again, thanks João!

**Comment by converge** [comment]
> > @fidencio tks for the review, just did another update following our chat.
> 
> @converge thanks a lot for the change.
> Would be possible to squash the second commit into the first one? I know, some repos do that while merging, but that's not the case for this project.
> 
> Anyways, after having this squashed, I think it's perfect and we can merge it after a second review / approval.
> 
> Again, thanks João!

Perfect, thanks a lot for all the information and patient.

**Review by jcvenegas — APPROVED** [review]

**Pull request merged**
